appveyor.yml

```diff
@@ -0,0 +1,41 @@
+environment:
+  matrix:
+  - julia_version: 1.0  # latest LTS
+  - julia_version: 1  # latest stable
+  - julia_version: nightly
+
+platform:
+  - x86 # 32-bit
```

Comment:
Double testing on 32-bit again. I would just drop it from AppVeyor and do Linux 32-bit on Travis

Reply:
Ah this time is is because I started from scratch in with AV
```suggestion
```